src/main/resources/application.properties

```diff
@@ -83,6 +83,7 @@ google.projectCreateTimeoutSeconds=600
 google.allowReuseExistingBuckets=false
 google.allowReuseExistingProjects=false
 google.firestoreRetries=10
+google.secureFolderResourceId=753276429356
```

Comment:
BTW, if you've got this default value here, you don't need this PR: https://github.com/broadinstitute/datarepo-helm-definitions/pull/428

Reply:
(It doesn't hurt, you just don't need it)

Reply:
You need it if your testing locally.

Reply:
really? Like you need this or the other PR?

Reply:
You need this if your testing against a local `bootRun`

Reply:
I'm surprised.  You should be able to do it with an environment variable (`GOOGLE_SECUREFOLDERRESOURCEID=...`) should do it